OctoKit.swift.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+  s.name             = "OctoKit.swift"
+  s.version          = "0.1.0"
+  s.summary          = "Swift version of the Octokit family"
+  s.description      = <<-DESC
+                        You are looking at the Swift port of the
+                        Octokit family. This is very unofficial and not maintained
+                        by Github.
+  DESC
+  s.homepage         = "https://github.com/nerdishbynature/octokit.swift"
+  s.license          = "MIT"
```

Comment:
Can you link to to the file as well? I missed that in the last PR https://github.com/nerdishbynature/RequestKit/commit/6c1839ec97007b7380f1e45c86507e82ef4aa276